src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -269,11 +268,19 @@ internal enum InterpolatedStringKind
             /// <summary>
             /// Normal interpolated string that just starts with $"
             /// </summary>
-            Normal,
+            Normal = 1,
             /// <summary>
             /// Verbatim interpolated string that starts with $@" or @$"
             /// </summary>
-            Verbatim,
+            Verbatim = 2,
+            /// <summary>
+            /// Single-line raw interpolated string that starts with some number of $ and at least three """.
```

Comment:
>some number of $ [](http://example.com/codeflow?start=69&length=16)

`at least one '$' and at least three '"'`? (also applies on line 281 below)

Reply:
Done!